internal/modules/auth/query.py

```diff
@@ -1,25 +1,67 @@
-import argparse, os, json
-
-from utils import exec_command
+"""
+This module consists of query commands for auth cosmos-sdk module
+"""
+import argparse
+import os
+import sys
+from internal.utils import exec_command
 
 DAEMON = os.getenv("DAEMON")
 RPC = os.getenv("RPC")
 
-# account_type is an user-defined data type used to validate account addresses read from Argument Parser.
+
 def account_type(address):
+    """account type is an user-defined data type used
+    to validate account addresses read from Argument Parser.
+
+    Args:
+        address (_str_): bech32 address to be queried.
+
+    Raises:
+        argparse.ArgumentTypeError: This Error is raised if the bech32 address is not valid.
+
+    Returns:
+        _tuple_: (boolean, json|str)
+    """
     status, response = query_account(address)
     if not status:
         raise argparse.ArgumentTypeError(response)
     return address
 
 
-# query_account function will take the bech32 address as input and output the information of account.
 def query_account(address):
-    try:
-        command = f"{DAEMON} query auth account {address} --node {RPC} --output json"
-        stdout, stderr = exec_command(command)
-        if len(stderr):
-            return False, stderr
-        return True, json.loads(stdout)
-    except Exception as e:
-        return False, e
+    """query account function will take the bech32 address
+    as input and output the information of account.
+
+    Args:
+        address (_str_): bech32 address to be queried.
+
+    Returns:
+        _tuple_: (bool, json|str)
+    """
+    command = f"{DAEMON} query auth account {address} --node {RPC} --output json"
+    return exec_command(command)
+
+
+def get_sequences(sender, receiver):
```

Comment:
There should be just single `query_sequence` method and we should call it multiple times as needed. Let's not send multiple addresses as args here and avoid multiple queries inside

Reply:
Oh looks like there's already `query_account` method. Let's just use that

Reply:
Okay.

Reply:
Okay using `query_account` function for fetching `sequence`